test/changed-files-test.js

```diff
@@ -156,6 +156,165 @@ describe(changedFiles, function() {
     ]);
   });
 
+  it('filters globs only', async function() {
+    fixturify.writeSync(tmpPath, {
+      'packages': {
+        'package-a': {
+          'changed.txt': 'test',
+          '.my-config.json': 'test',
+        },
+        'changed-without-config-1.txt': 'test',
+      },
+      'changed-without-config-2.txt': 'test',
+      '.my-config.json': 'test',
+    });
+
+    await execa('git', ['add', '.'], { cwd: tmpPath });
+    await execa('git', ['commit', '-m', 'test'], { cwd: tmpPath });
+
+    let _changedFiles = await changedFiles({
+      cwd: tmpPath,
+      silent: true,
+      globs: ['**/.my-config.json'],
+    });
+
+    expect(_changedFiles).to.deep.equal([
+      'packages/package-a/.my-config.json',
+      '.my-config.json',
+    ]);
+  });
+
+  it('globs top level vs nested', async function() {
```

Comment:
I think we should remove 'globs top level vs nested', 'globs brace expansion', and 'globs star matches'. We are duplicating minimatch's own test suite, and slowing down our own test suite. Thoughts?

Reply:
Yeah, I was thinking that it's pretty much a duplicate and not that useful as it does not catch everything minimatch can do. The idea was more in line to "keep the contract that `globs` will always accept minimatch syntax". Which again is not guaranteed by these.

Will remove.